tests/ZfcRbacTest/Factory/AuthorizationServiceDelegatorTest.php

```diff
@@ -73,6 +74,10 @@ public function testAuthorizationServiceIsInjectedWithDelegator()
     {
         $serviceManager = ServiceManagerFactory::getServiceManager();
 
+        if (method_exists($serviceManager, 'build')) {
```

Comment:
this is to check whether servicemanager v2 oder v3 is used.
i would have used https://github.com/Ocramius/PackageVersions, but this is php7 only.